doc/api/modules.md

```diff
@@ -850,6 +850,21 @@ by the [module wrapper][]. To access it, require the `Module` module:
 const builtin = require('module').builtinModules;
 ```
 
+### module.makeRequire(filename)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filename` {string|URL} Filename for relative importing
+* Returns: {[`require`][]} Require function
```

Comment:
Maybe it would be more consistent with other cases to add this to type parser between [other Node.js "types"](https://github.com/nodejs/node/blob/cd7d7b15c1eeccfe2facdf9a671034d93b6bf467/tools/doc/type-parser.js#L32-L94). But I am not sure if we should do this for all solitary cases.

Reply:
worth a separate pr?
```
doc/api/modules.md
873:[`require`]: #modules_require

doc/api/addons.md
1177:[require]: modules.html#modules_require

doc/api/globals.md
168:[`require()`]: modules.html#modules_require
```